frog/imports/api/generateReactiveFn.js

```diff
@@ -189,10 +189,11 @@ export class Doc {
     );
     if (LIData.liType === 'li-activity') {
       // In this case the activity also needs to be duplicated
+      const ac = connection.get('rz', LIData.payload.rz);
+      ac.fetch();
```

Comment:
This is asynchronous, so you will probably have timing issues here in production (not on localhost). You should use a callback or Promise). See other places where we use connection.get - we always wait for onload etc.

Reply:
Now using await function